networkx/algorithms/matching.py

```diff
@@ -33,6 +33,12 @@ def maximal_matching(G):
     matching : set
         A maximal matching of the graph.
 
+    Examples
+    --------
+    >>> G=nx.Graph ([(1, 2), (1, 3), (2, 3), (2, 4), (3, 5), (4, 5)])
```

Comment:
It looks like some code formatting issues are still sneaking through - the best way to ensure this doesn't happen is to install the pre-commit hooks in your development environment, e.g. `pip install -r requirements/dev.txt` and then `pre-commit install`.

Reply:
Okay sir, thanks. I have pre-commit installed but  I don't know why I still have formatting issues sneaking through.